Sileo/UI/PackageViewController/DepictionView/DepictionStackView/DepictionStackView.swift

```diff
@@ -27,21 +27,22 @@ class DepictionStackView: DepictionBaseView {
                 isLandscape = true
             }
         }
-
+        
         for viewDict in views {
             guard (viewDict["class"] as? String) != nil else {
                 return nil
             }
         }
-
+        
         super.init(dictionary: dictionary, viewController: viewController, tintColor: tintColor, isActionable: isActionable)
         for viewDict in views {
-            if let view = DepictionBaseView.view(dictionary: viewDict, viewController: viewController, tintColor: tintColor, isActionable: isActionable) {
+            let view = DepictionBaseView.view(dictionary: viewDict, viewController: viewController, tintColor: tintColor, isActionable: isActionable)
+            if let view = view {
```

Comment:
Set the original over 2 lines instead of separating to 2 statements

Reply:
![image](https://user-images.githubusercontent.com/26681721/111655120-49d56a00-8801-11eb-8952-c1d38f5ecea3.jpeg)

Reply:
Done